docs/HowToUseSmartFSDumpVisualizer.md

```diff
@@ -21,11 +21,15 @@ After downloading the SDV release (zip/tar.gz) from the following links,
 Decompress the release file and execute SDV.exe/SDV.
 
 
-## Example of Multi-heap Usage
+## Example of SDV Usage
 After starting the tool, an initial screen is shown like below.  
 ![Initial](./media/Initial_screen.png)
 
-The contents of a smartfs dump file can be shown by opening it through "open" menu.
+The contents of a smartfs dump file* can be shown by opening it through "open" menu.
+(*Note that now dump files can be obtained only though tools provided by board manufacturers.
```

Comment:
By documentation rules, you should format the note as shown below:

```
> **NOTE** blah blash
> blah blah
```

Reply:
Updated, thank you!

Reply:
You missed `>`. Current is wrong.

Reply:
Updated. Thank you again!